.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -15,5 +15,9 @@ A clear and concise description of what you want to happen.
 **Describe alternatives you've considered**
 A clear and concise description of any alternative solutions or features you've considered.
 
+**Priority:** Priority level of this bug [critical, high, medium, low]
+
+**Stakeholders:** Concerned stakeholder(s) of this bug [CMS, FactoryOps, Fermilab, FIFE, IGWN, OSG]
```

Comment:
Like before I'd add an "Info" section with Priority and Stakeholders as items

Reply:
Updated as per the comment above